AbletonMCP_Remote_Script/handlers/clips.py

```
@@ -0,0 +1,726 @@
+
+
+def set_clip_looping(song, track_index, clip_index, looping, ctrl=None):
+    """Set the looping state of a clip."""
+    try:
+        _, clip = get_clip(song, track_index, clip_index)
+        clip.looping = looping
+        return {
+            "track_index": track_index,
+            "clip_index": clip_index,
+            "looping": clip.looping,
+        }
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)